cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go

```diff
@@ -259,7 +259,8 @@ func (client *autoscalingGceClientV1) DeleteInstances(migRef GceRef, instances [
 		req.Instances = append(req.Instances, GenerateInstanceUrl(i))
 	}
 	op, err := client.gceService.InstanceGroupManagers.DeleteInstances(migRef.Project, migRef.Zone, migRef.Name, &req).Do()
-	if err != nil {
+	wasConflictErr := op != nil && op.HttpErrorStatusCode == http.StatusConflict
+	if !wasConflictErr && err != nil {
 		return err
```

Comment:
Can you move `wasConflictErr` var inside the `if err != nil` condition & log a warning whenever this happens? CA generally shouldn't attempt to delete the same VM twice so it would be good to at least leave a trace that this happened.

Reply:
[082e961](https://github.com/kubernetes/autoscaler/pull/5192/commits/082e961c6b5cd09407ec92ff199c569edb73e4fe)